pkg/ephemeral-disk-utils/utils.go

```diff
@@ -66,13 +65,13 @@ func (om *OwnershipManager) SetFileOwnership(file string) error {
 	return os.Chown(file, uid, gid)
 }
 
-func RemoveFile(path string) error {
-	err := os.RemoveAll(path)
-	if err != nil && os.IsNotExist(err) {
-		return nil
-	} else if err != nil {
-		log.Log.Reason(err).Errorf("failed to remove %s", path)
-		return err
+func RemoveFilesIfExist(paths ...string) error {
+	var err error
+	for _, path := range paths {
+		err = os.Remove(path)
+		if err != nil && !os.IsNotExist(err) {
+			return err
```

Comment:
@dankenigsberg This function is used for cleanup. Maybe we want to try and remove all the files even if we got an error on one of them , and return an error if any of the removal failed, what do you think?

Reply:
I can be more greedy, but it would make this function (slightly) more complicated. A failure to remove an existing path in our use case is sooooo rare (I even failed to simulate it in the unit tests, because it is run by root) that it should be treated like an end-of-the-world scenario.

I'd add it if you insist.

Reply:
@dankenigsberg NP, Especially that I see it is also called from another place.  Just keep in mind that if you get an error on the first file, you may leave some garbage other files seating around. 
Maybe we should add special purpose cleaning functions in a different PR.